kevm_pyk/src/kevm_pyk/utils.py

```diff
@@ -17,6 +21,24 @@
     splitConfigFrom,
     substitute,
 )
+from pyk.ktool import KPrint
+from pyk.ktool.kprint import build_symbol_table
+from pyk.utils import hash_str
+
+
+def KPrint_make_unparsing(_self: KPrint, extra_modules: List[KFlatModule] = []) -> KPrint:
```

Comment:
```suggestion
def KPrint_add_unparsing_module(_self: KPrint, extra_modules: List[KFlatModule] = []) -> KPrint:
```

Reply:
It can be renamed when it's upstreamed. I'm sure it will change and get less hacky over time.